apps/website/netlify/edge-functions/strangler.ts

```diff
@@ -0,0 +1,85 @@
+import { Config, Context } from 'https://edge.netlify.com';
+
+type LegacyHost = {
+  url: string;
+  urlFilter?: (url: URL) => boolean;
```

Comment:
How about making it more abvious?
```ts
applies: (url: URL) => boolean;
```
And then we set it to `applies: () => true` in the drupal host?